TEST_APPS/testcases/netsocket/TCPSOCKET_ACCEPT.py

```diff
@@ -0,0 +1,102 @@
+"""
+Copyright 2018 ARM Limited
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import threading
+import time
+
+from mbed_clitest.bench import Bench
+from interface import interfaceUp, interfaceDown
+from mbed_clitest.tools import test_case
+from mbed_clitest.TestStepError import SkippedTestcaseException
+
+class Testcase(Bench):
+    def __init__(self):
+        Bench.__init__(self,
+                       name="TCPSOCKET_ACCEPT",
+                       title = "TCPSOCKET_ACCEPT",
+                       purpose = "Test that TCPSocket::bind(), TCPSocket::listen() and TCPSocket::accept() works",
+                       status = "released",
+                       component= ["mbed-os", "netsocket"],
+                       author = "Juha Ylinen <juha.ylinen@arm.com>",
```

Comment:
Don't copy&paste the author line.. You can remove the whole line. It is not used anywhere.